### ISSUE TITLE
accordion - accessibility issue 

### ISSUE THREAD

**Issue by UiJo** [Promact/md2#301]
#### Bug, feature request, or proposal:
Violation `md2-accordion > md2-accordion-tab[role="accordion-tab"]:nth-child(1)`
when testing the accordion component for a11y (using Lighthause or axe)

#### What is the expected behavior?
No violation

#### What is the current behavior?
Violation `md2-accordion > md2-accordion-tab[role="accordion-tab"]:nth-child(1)`

#### What are the steps to reproduce?
Install Lighthouse or axe Chrome plugin and run a verification for http://code.promactinfo.com/md2/#/accordion

#### What is the use-case or motivation for changing an existing behavior?
This component should be a11y compliant, just like the Angular Material accordion.

#### Which versions of Angular, MD2, OS, browsers are affected?
tested with Chrome, Lighthouse and the latest version of Material Design 2 (0.0.29)
